vocab.py

```diff
@@ -23,4 +23,5 @@ class ValidDataSourceName(Enum):
     """
     
     CANVAS_API = auto()
+    UNIZIN_DATA_PLATFORM = auto()
```

Comment:
Do you think this should reference BigQuery? If we need to use the context store in the future, things might get confusing.

Reply:
Not BigQuery.  Maybe `UNIZIN_DATA_PLATFORM_EVENTS`?

Reply:
I'll leave it up to you.